### DISCUSSION TITLE
nerdctl is not listing images (`nerdctl -n moby  images`)

### DISCUSSION THREAD

**Discussion by kesavkolla** [containerd/nerdctl#1667] [Q&A] [answered]
I have docker installed on the same machine as nerdctl. I have used docker pull to pull image. 

I have the following output

```
docker images
REPOSITORY   TAG       IMAGE ID       CREATED       SIZE
alpine       latest    49176f190c7e   3 weeks ago   7.05MB
```

I was expecting similar output when I use nerdctl. But I get empty results:

```
nerdctl -n moby  images
REPOSITORY    TAG    IMAGE ID    CREATED    PLATFORM    SIZE    BLOB SIZE
```

I have custom docker root directory:

```
docker info | grep Root 
Docker Root Dir: /data/docker
```

Is there any setting I have to use for nerdctl to get the same output as docker?

**Comment by AkihiroSuda** [comment, marked as answer]
Expected behavior, because Docker does not use the containerd image store.
Will be changed in Docker v24 though. ( https://github.com/moby/moby/issues?q=+label%3Acontainerd-integration+ )
